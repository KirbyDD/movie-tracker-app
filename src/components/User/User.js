import React from 'react';
import ShowFavorites from '../../Containers/ShowFavorites/ShowFavorite'

const User = () => {
  return(
    <section>
      <ShowFavorites />>
    </section>
  )
}

export default User;