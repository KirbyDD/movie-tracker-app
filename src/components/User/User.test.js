import React from 'react';
import {shallow} from 'enzyme';
import User from './User';

describe('Movie Container', () => {
  let wrapper;
  beforeEach(function() {
    wrapper = shallow(
      <User />
    )
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})