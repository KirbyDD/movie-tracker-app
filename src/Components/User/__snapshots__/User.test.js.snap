// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Container should match snapshot 1`] = `
<section>
  <ShowFavorites />
  &gt;
</section>
`;
